apps/coscrad-frontend/src/components/transcripts/transcript-presenter.tsx

```diff
@@ -0,0 +1,50 @@
+import { ITranscriptItem, ITranscriptParticipant } from '@coscrad/api-interfaces';
+import { isNullOrUndefined } from '@coscrad/validation-constraints';
+import { SubtitlesRounded } from '@mui/icons-material';
+import { Box, Card, CardContent, CardHeader, Divider, Typography } from '@mui/material';
+
+export const TranscriptPresenter = ({ transcript }): JSX.Element => {
+    return (
+        <Card elevation={0}>
+            <CardHeader
+                avatar={<SubtitlesRounded color="primary" />}
+                title={
+                    <Typography variant="h4" margin={'auto 0'}>
+                        Transcript
+                    </Typography>
+                }
+            />
+            <CardContent>
+                {isNullOrUndefined(transcript)
+                    ? null
+                    : transcript.participants.map((item: ITranscriptParticipant) => (
+                          <Box>
+                              <Typography variant="body1" sx={{ fontWeight: 'bold' }}>
+                                  Participants:&nbsp;
+                              </Typography>
+                              {item.name} ({item.initials})
+                          </Box>
+                      ))}
+
+                <Divider sx={{ marginY: 2 }} />
+
+                {isNullOrUndefined(transcript)
+                    ? null
```

Comment:
I think this could be written as
```{!isNullOrUndefined(transcript) && .....}```
So just check that the transcript is not null or undefined and if not, map through the transcript.

Reply:
No it cannot @yaanahuu2 . Then you will render `false`  as a react node. I have to emphasize yet again that we want to explicitly return `null` but not `undefined` or `false` as react nodes. This may be a use case for our `<Optional>` abstraction.

Reply:
We've done this using Optional.

Reply:
Note that there is a structural issue with `<Optional>`, namely we want to lazily execute the rendering of the component only if the `predicateValue` is `true`. So we decided the original code was the simplest and clearest.